src/index.js

```diff
@@ -0,0 +1,82 @@
+// eslint-disable-next-line no-unused-vars
+import _ from 'lodash';
+import './style.css';
+import checkbox from './completed.js';
+import newTask, {
+  enterToAddTask,
+  editTask,
+  removeCompleted,
+} from './tasksArray.js';
+
+let todoItems = [];
+
+const lastItem = document.body.querySelector('.last-item');
+const listMain = document.body.querySelector('.todo-list');
+
+function tasksHTML() {
+  todoItems.forEach((task, i) => {
+    task.index = i;
+    i += 1;
+  });
```

Comment:
Can be extracted for clarity to a method called:

addIndexAttributeToTasks(todoItems)